cron/cron.ts

```diff
@@ -93,51 +94,25 @@ export async function daily(bot: Bot<BotContext>) {
       const keyboardText = HLanguage(user.language, 'mainKeyboard')
       const buttons = customKFunction(2, ...keyboardText)
 
-      if (now.getDay() == 5) {
-        bot.api
-          .sendPhoto(user.userId, file, {
-            caption: message + (randomHadith ? `\n\n${randomHadith.content}` : ''),
-            reply_markup: { keyboard: buttons.build(), resize_keyboard: true },
-          })
-          .then(() => {
-            user.status = true
-          })
-          .catch(async (error) => {
-            if (error.description == 'Forbidden: bot was blocked by the user') {
-              user.status = false
-            } else if (error.description == 'Forbidden: user is deactivated') {
-              await Model.User.findOneAndUpdate({ userId: user.userId }, { deletedAt: new Date() })
-            } else console.error('Error:', error)
-          })
-          .finally(() => {
-            user.save()
-          })
-      } else {
-        bot.api
-          .sendMessage(user.userId, message + (randomHadith ? `\n\n${randomHadith.content}` : ''), {
-            reply_markup: { keyboard: buttons.build(), resize_keyboard: true },
-          })
-          .then(() => {
-            user.status = true
-          })
-          .catch(async (error) => {
-            if (error.description == 'Forbidden: bot was blocked by the user') {
-              user.status = false
-            } else if (error.description == 'Forbidden: user is deactivated') {
-              await Model.User.findOneAndUpdate({ userId: user.userId }, { deletedAt: new Date() })
-            } else console.error('Error:', error)
-          })
-          .finally(() => {
-            user.save()
-          })
+      // if (weekDay == 5) {
+      //   await bot.api
+      //     .sendPhoto(user.userId, file, {
+      //       caption: message + (randomHadith ? `\n\n${randomHadith.content}` : ''),
+      //       reply_markup: { keyboard: buttons.build(), resize_keyboard: true },
+      //     })
+      // } else {
+      try {
+        await bot.api.sendMessage(user.userId, message + (randomHadith ? `\n\n${randomHadith.content}` : ''), {
+          reply_markup: { keyboard: buttons.build(), resize_keyboard: true },
+        })
```

Comment:
wuni tegidan statusini true qib qo'yiw keremasmi

Reply:
could you explain why it was set to false

Reply:
agar block qigan bo'sa status = false, agar succes bo'sa status = true ya'ni bilbo'midiku uni status oldin false bo'gan boliwi mumkinde

Reply:
is it right time to meeting